### PR TITLE
kamid: init at 0.1

### PR DESCRIPTION
**Pull request by heph2**
###### Motivation for this change
kamid is a FREE, easy-to-use and portable implementation of a 9p file server daemon for UNIX-like systems.
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/ftp/kamid/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+, fetchurl
+, readline
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kamid";
+  version = "0.1";
+
+  src = fetchurl {
+    url = "https://github.com/omar-polo/kamid/releases/download/0.1/${pname}-${version}.tar.gz";
```
Comment (bobby285271) [inline pkgs/servers/ftp/kamid/default.nix:16]:
```suggestion
    url = "https://github.com/omar-polo/kamid/releases/download/${version}/${pname}-${version}.tar.gz";
```

**Review by bobby285271 — COMMENTED** [review]
Please also add your package to `pkgs/top-level/all-packages.nix`.

**Comment by heph2** [comment]
> Please also add your package to `pkgs/top-level/all-packages.nix`.

Added! :D Should i squash it into "kamid: init at 0.1" or is it fine?

**Comment by Lassulus** [comment]
Result of `nixpkgs-review pr 162739` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>kamid</li>
  </ul>
</details>

**Comment by Lassulus** [comment]
> > Please also add your package to `pkgs/top-level/all-packages.nix`.
> 
> Added! :D Should i squash it into "kamid: init at 0.1" or is it fine?

i have squashed it while merging, generally  you shold squash it, sometimes mergers forget to check it  :)

**Pull request merged**